lib/assertions/same.test.js

```diff
@@ -1,80 +1,95 @@
 "use strict";
 
-var testHelper = require("../test-helper");
+var assert = require("assert");
+var referee = require("../referee");
 
 var obj = { id: 42 };
 var obj2 = { id: 42 };
 
-testHelper.assertionTests("assert", "same", function(pass, fail, msg, error) {
-    pass("when comparing object to itself", obj, obj);
-    fail("when comparing different objects", obj, obj2);
-    pass("when comparing strings", "Hey", "Hey");
-    pass("when comparing booleans", true, true);
-    pass("when comparing numbers", 32, 32);
-    pass("when comparing infinity", Infinity, Infinity);
-    fail("when comparing without coercion", 666, "666");
-    fail("when comparing falsy values without coercion", 0, "");
-    pass("when comparing null to null", null, null);
-    pass("when comparing undefined to undefined", undefined, undefined);
-    msg(
-        "include objects in message",
-        "[assert.same] Obj expected to be the same object as {  }",
-        "Obj",
-        {}
-    );
-    msg(
-        "include custom message",
-        "[assert.same] Back again: Obj expected to be the same object as {  }",
-        "Obj",
-        {},
-        "Back again"
-    );
-    pass("when comparing NaN to NaN", NaN, NaN);
-    fail("when comparing -0 to +0", -0, +0);
-    error(
-        "when comparing strings",
-        {
-            code: "ERR_ASSERTION",
-            actual: "foo",
-            expected: "bar",
-            operator: "assert.same"
-        },
-        "foo",
-        "bar"
-    );
-});
+describe("assert.sameTest", function() {
```

Comment:
I think this should read `assert.same`, to follow the pattern from the other files

Reply:
thanks